deploy/terraform/modules/hdb_node/vm-hdb.tf

```
@@ -127,6 +127,7 @@ resource "azurerm_availability_set" "hana-as" {
   resource_group_name          = var.resource-group[0].name
   platform_update_domain_count = 20
   platform_fault_domain_count  = 2
+  proximity_placement_group_id =  lookup(var.infrastructure, "ppg", false) != false ? (var.ppg[0].id) : null
```

There are two spaces after =:
```suggestion
  proximity_placement_group_id = lookup(var.infrastructure, "ppg", false) != false ? (var.ppg[0].id) : null
```